### PR TITLE
Checkbox

### PR DESCRIPTION
**Pull request by SebaMutuku**

### CODE REVIEW

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java [hunk 1]
```diff
@@ -144,6 +144,21 @@ public void onReceive(Context context, Intent intent) {
...
+            JSONObject selectedOption = formUtils.getOptionFromOptionsUsingKey(options, value);
+            newValue.put(JsonFormConstants.VALUE, value);
+            newValue.put(JsonFormConstants.TEXT, selectedOption.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
+            return newValue;
+        }
+        newValue.put(JsonFormConstants.VALUE, value);
+        newValue.put(JsonFormConstants.TEXT, item.optString(JsonFormConstants.TRANSLATION_TEXT, ""));
+        return newValue;
+    }
+
```
Comment (dubdabasoduba) [inline android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java:161]: Isn't this method already in the Utils? Please reuse that

Comment (dubdabasoduba) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java:161]: If not please move it to the Utils

Comment (dubdabasoduba) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java:161]: Also add Java Docs for any public methods you write

**Review by dubdabasoduba — CHANGES_REQUESTED** [review]

File: android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java (hunk 1 above)
Comment (SebaMutuku) [reply android-json-form-wizard/src/main/java/com/vijay/jsonwizard/activities/JsonFormActivity.java:161]: Hi, this was moved to utils in the new widgets touched. The function is already in Utils

**Review by dubdabasoduba — APPROVED** [review]

**Pull request merged**